src/highdicom/pr/sop.py

```diff
@@ -1409,93 +770,122 @@ def __init__(
             of `highdicom.base.SOPClass`
 
         """  # noqa: E501
+        for kw in [
+            'rescale_intercept',
+            'rescale_slope',
+            'rescale_type',
+            'modality_lut',
+            'softcopy_voi_luts',
+        ]:
+            if kw in kwargs:
+                raise TypeError(
+                    'ColorSoftcopyPresentationState() got an unexpected '
+                    f'keyword argument "{kw}".'
+                )
```

Comment:
Like for the GSPS case I mentioned above, this test is no longer needed and should be removed

Reply:
Removed via a004405bfb943ef2cb63d6aaf52cba79fb227d21